dist/chartjs-plugin-piechart-outlabels.esm.js

```diff
@@ -659,25 +683,16 @@ var plugin = {
   afterDatasetDraw: function(chart, args) {
     var elements = args.meta.data || [];
     var ctx = chart.ctx;
-    var el, label, index;
-
-    for (var i = 0; i < 2 * elements.length; ++i) {
-      index = i < elements.length ? i : i - elements.length;
 
-      el = elements[index];
-      label = el[LABEL_KEY];
-      if (!label) {
-        continue;
-      }
-
-      if (i < elements.length) {
-        label.update(el, elements, i);
-        label.drawLine(ctx);
-      } else {
-        label.draw(ctx);
+    elements.forEach((el, index) => {
+      const outlabelPlugin = el[PLUGIN_KEY];
+      if (!outlabelPlugin) {
+        return;
       }
-    }
-  }
+      outlabelPlugin.update(el, elements, index);
```

Comment:
Do you need to call `update` here since it's already called in the `afterUpdate` hook?

Reply:
Yes to have the right sizes after the resize of the inner and outer radius